### PR TITLE
Update findAllMatches to return an array of objects

### PR DESCRIPTION
**Pull request by Obleo33**

### CODE REVIEW

File: src/helper.js [hunk 1]
```diff
@@ -37,9 +35,9 @@ export default class DistrictRepository {
   findAllMatches(input) {
     const locationKeys = Object.keys(this.data);
     if(input) {
-      return locationKeys.filter(location => location.toLowerCase().includes(input.toLowerCase()))
+      const search = locationKeys.filter(location => location.toLowerCase().includes(input.toLowerCase()))
+      return search.map((location,index)=> search[index] = this.data[location])
```
Comment (Obleo33) [inline src/helper.js:39]: This map function will now return an array of objects

File: src/helper.js [hunk 2]
```diff
@@ -37,9 +35,9 @@ export default class DistrictRepository {
...
     }
-
-    return locationKeys
+    return locationKeys.map((location,index)=> this.data[location])
```
Comment (Obleo33) [inline src/helper.js:41]: will return an array of objects

Comment (becs919) [reply src/helper.js:41]: Nice! Im glad Taylor sorted us out on this.

File: src/helper.js (hunk 1 above)
Comment (becs919) [reply src/helper.js:39]: ditto to my other comment

**Comment by becs919** [comment]
Everything looks good! Ill work on rending to page!

**Pull request merged**